.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -9,3 +9,4 @@ Fixes #
 ℹ Please replace this with a clear and concise description of the steps required to validate this pull request.
 
 - [ ] Please check here if your pull request includes additional test coverage.
+- [ ] I have considered updating the podspec version.
```

Comment:
What do you think of this wording?

```suggestion
- [ ] I have considered updating the `version` in the `.podspec` file.
```

IMHO, it makes it clearer that you need to open the file and edit the `version` field

Reply:
I think that's a great suggestion, accepted!